tests/cli/test_worker.py

```diff
@@ -1,10 +1,97 @@
+import asyncio
+import inspect
+import logging
+
+import pytest
 from typer.testing import CliRunner
 
 from docket.cli import app
+from docket.docket import Docket
+from docket.worker import Worker
+
+
+def test_worker_command(
+    runner: CliRunner,
+    docket: Docket,
+    caplog: pytest.LogCaptureFixture,
+):
+    """Should run a worker until there are no more tasks to process"""
+    with caplog.at_level(logging.INFO):
+        result = runner.invoke(
+            app,
+            [
+                "worker",
+                "--until-finished",
+                "--url",
+                docket.url,
+                "--docket",
+                docket.name,
+            ],
+        )
+        assert result.exit_code == 0
+
+    assert "Starting worker" in caplog.text
+    assert "trace" in caplog.text
+
+
+def test_worker_command_exposes_all_the_options_of_worker():
```

Comment:
@jakekaplan I added this test to make sure we're always exposing all the arguments in the CLI so we can keep things synced up